Dev 3.11

GigaZiv opened the pull request:
Выполнено задание на 18 спринт

С уважением Жуков Игорь

GigaZiv committed:
Dev 3.11

Выполнено задание на 18 спринт

С уважением Жуков Игорь

kosstyango commented:
⚠️ Критические замечания:
   В темной теме заглушки "Вы не создали ни одного плейлиста" и "Ваша медиатека пуста" показывают картинку с черной нотой на черном фоне - это не соответствует дизайну (см.скрин).
![Спринт_18_Berholz_1](https://github.com/user-attachments/assets/e1149230-ba21-40d8-b49c-ba98ab89ea77)


🍏 Рекомендации, не влияющие на результат проверки работы:
   Дефект из 9 спринта: треки с ОЧЕНЬ длинным названием альбома отображаются неправильно - без указания длительности трека. Проверь поиском слова "метелица" - там таких примеров будет четыре (см.скрин).
![Спринт_18_Berholz_2](https://github.com/user-attachments/assets/e237cb0c-b3e7-4cd4-b6a6-f0877a1c4ede)

GigaZiv committed:
Dev 3.12

1. Исправлена "Тема" во вкладках;
2. Исправлено поведение с длинными наименованиями;

С уважением Жуков Игорь
